README.md

```diff
@@ -94,16 +94,10 @@ add flynt to your [pre-commit](https://www.pre-commit.com) hooks.
 
 Add a new section to `.pre-commit-config.yaml`:
 ```
-   - repo: local
-     hooks:
-         - id: flynt
-           name: flynt
-           entry: flynt
-           args: [--fail-on-change]
```

Comment:
why is this line redundant? Default is false for `fail-on-change`

Reply:
Good catch, I didn't notice at first. This is explained in pre-commit's doc [here](https://pre-commit.com/#creating-new-hooks)
> The hook must exit nonzero on failure or modify files.

so `--fail-on-change` is implicit in the case of a hook !

Reply:
What do you mean with `implicit`? My  understanding is that flynt will run, but with `fail-on-change` == False, meaning it will always return 0 and actually modify  files.

So Pre-commit recognizes file modification itself  and there is  no need for  non-zero return value? this means --fail-on-change is redundant for  the purpose of  pre-commit hooks?

Reply:
Your understanding is correct. It actually modifies files in place (which is what we want !) and yes, fail-on-change is redundant here (which is what I meant by "implicit", sorry I chose this incorrect wording)